issm-project-week1/src/index.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./App";
+import "./style.css";
+
+// const num = [1, 2, 3, 4, 5];
```

Comment:
Always remove unused code and unnecessary comment before requesting merge

Reply:
It's kept for future reference